File: dotnet/githerb/Utils/HerbsSeeder.cs
public class HerbsSeeder
{
    public static List<Herb> SeedHerbs()
    {
        // Generate 10 herbs using real herb names using json format
        var herbsJson = """
                [
                    { "Id": 1, "Name": "Basil" },
                    { "Id": 2, "Name": "Rosemary" },
                    { "Id": 3, "Name": "Thyme" },
                    { "Id": 4, "Name": "Sage" },
                    { "Id": 5, "Name": "Mint" },
                    { "Id": 6, "Name": "Parsley" },
                    { "Id": 7, "Name": "Cilantro" },
                    { "Id": 8, "Name": "Dill" },
                    { "Id": 9, "Name": "Oregano" },
                    { "Id": 10, "Name": "Chives" }
                ]
                """;

        return System.Text.Json.JsonSerializer.Deserialize<List<Herb>>(herbsJson) ?? new List<Herb>();
    }
}

File: python/utils/herbs_seeder.py
import json
from models.herb import Herb
from utils.in_memory_datastore import herbs_data_store

# Seeder JSON string
herbs_json = """
            [
                { "id": 1, "name": "Basil" },
                { "id": 2, "name": "Rosemary" },
                { "id": 3, "name": "Thyme" },
                { "id": 4, "name": "Sage" },
                { "id": 5, "name": "Mint" },
                { "id": 6, "name": "Parsley" },
                { "id": 7, "name": "Cilantro" },
                { "id": 8, "name": "Dill" },
                { "id": 9, "name": "Oregano" },
                { "id": 10, "name": "Chives" }
            ]
            """

def seed_herbs():
    # Parse the JSON string
    herbs = json.loads(herbs_json)
    # Clear existing data
    herbs_data_store.clear()
    # Populate the data store
    for herb in herbs:
        herbs_data_store.append(Herb(**herb))